gson/src/main/java/com/google/gson/JsonDeserializer.java

```diff
@@ -51,7 +51,9 @@
  * class IdDeserializer implements JsonDeserializer&lt;Id&gt;() {
```

Comment:
Could you please also adjust this here to fix #626 (the description of that issue is not very great but it seems to be about this)?
```suggestion
 * class IdDeserializer implements JsonDeserializer&lt;Id&gt; {
```

Reply:
updated